src/ApplicationPaths.ts

```diff
@@ -62,6 +62,7 @@ export function applicationPaths({
 		"/calendar": calendar,
 		"/calendar/:view": calendar,
 		"/calendar/:view/:date": calendar,
+		"/calendar/:view/:date/:time": calendar,
```

Comment:
I don't think the time should be in the URL.

We can't select the time, it is not something that you can change and it should always be the current time.

We should decide about `shouldScroll` not in the URL, it's not responsibility of routing to know to scroll to current time or not.